packages/e2e-test/src/commands/run.ts

```diff
@@ -427,24 +430,42 @@ async function testAppServe(pluginId: string, appDir: string) {
 }
 
 /** Drops PG databases */
-async function dropDB(database: string) {
-  const config = {
-    host: process.env.POSTGRES_HOST,
-    port: process.env.POSTGRES_PORT,
-    user: process.env.POSTGRES_USER,
-    password: process.env.POSTGRES_PASSWORD,
-  };
-
+async function dropDB(database: string, client: string) {
   try {
-    await pgtools.dropdb(config, database);
+    if (client === 'postgres') {
+      const config = {
+        host: process.env.POSTGRES_HOST ? process.env.POSTGRES_HOST : '',
+        port: process.env.POSTGRES_PORT ? process.env.POSTGRES_PORT : '',
+        user: process.env.POSTGRES_USER ? process.env.POSTGRES_USER : '',
+        password: process.env.POSTGRES_PASSWORD
+          ? process.env.POSTGRES_PASSWORD
+          : '',
+      };
+      await pgtools.dropdb(config, database);
+    } else if (client === 'mysql') {
+      const connectionString =
+        process.env.BACKSTAGE_TEST_DATABASE_MYSQL8_CONNECTION_STRING ?? '';
+      const connection = await mysql.createConnection(connectionString);
+      await connection.execute('DROP DATABASE ?', [database]);
+    }
   } catch (_) {
-    /* do nothing*/
+    /* do nothing */
   }
 }
 
 /** Clean remnants from prior e2e runs */
-async function preCleanPostgres() {
+async function preCleanDatabase() {
   print('Dropping old DBs');
+  if (Boolean(process.env.POSTGRES_HOST)) {
+    await dropClientDatabases('postgres');
+  }
+  if (Boolean(process.env.BACKSTAGE_TEST_DATABASE_MYSQL8_CONNECTION_STRING)) {
```

Comment:
no need for using such a specific var - if you look at what the e2e test does today, it just inserts a variable name that you choose yourself https://github.com/backstage/backstage/blob/10d83db0d203c1aa88c7c86ac24b4275833fe401/.github/workflows/verify_e2e-linux.yml#L67

Just `MYSQL_CONNECTION` should do.

But on that topic, this doesn't actually get exercised by the e2e tests yet (the PR doesn't seem to have added the call using this to the workflows). That's intentional and to be done later right?